test/test_routing.py

```diff
@@ -2057,6 +2057,9 @@ def try_if(condition, status):
 
     # njs
 
+    if not option.available['modules']['njs']:
```

Comment:
`require({'modules': {'njs': 'any'}})` should mark test as skipped rather than terminating silently

Reply:
Doesn't that make it skip the whole test?

Maybe we need to split that test up...

Reply:
> Doesn't that make it skip the whole test?

No, only the tests after the `require()` call are skipped, but the entire test suite will be marked as skipped.

> Maybe we need to split that test up...

Ideally, yes. It would make the tests more explicit.

Reply:
Which is what I did, so it looks good to go...